README.rst

```diff
@@ -61,6 +61,30 @@ edX Logo with ®
   // Or the png version
   import logo from '@edx/brand/logo-trademark.png';
 
+edX Logo in white
+-----------------
+
+.. image:: /logo-white.svg
+    :alt: edX
+    :width: 128px
+
+.. code-block:: javascript
+
+  import logo from '@edx/brand/logo-white.svg';
+
+  // Or the png version
+  import logo from '@edx/brand/logo-white.png';
+
+edX Favicon
+-----------------
+
+.. image:: /favicon.ico
+    :alt: edX
+    :width: 128px
+
+.. code-block:: javascript
+
+  // @edx/brand/favicon.ico;
```

Comment:
Is this intentionally left here and commented out?

Reply:
Yea, since I'm not positive how people will consume it, but I still wanted it in a code block